src/endpoints/querier/main.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"context"
+	"encoding/xml"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptrace"
+	"net/url"
+	"os"
+	"path"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"endpoints/fetcher"
+	"fhir"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Metrics collected inside built-in prometheus vector.
+// Each METRIC has its own registrations
+var httpCodesGaugeVec *prometheus.GaugeVec
+var responseTimeGaugeVec *prometheus.GaugeVec
+var tlsVersionGaugeVec *prometheus.GaugeVec
+var fhirVersionGaugeVec *prometheus.GaugeVec
+var totalUptimeChecksCounterVec *prometheus.CounterVec
+var totalFailedUptimeChecksCounterVec *prometheus.CounterVec
+
+// Need to define timeout or else it is infinite
+var netClient = &http.Client{
+	Timeout: time.Second * 35,
+}
+
+func getHTTPRequestTimingFor(urlString string, organizationName string, recordLongRunningMetrics bool) {
+	// recover from fatal errors
+	if err := recover(); err != nil {
+		println(err)
+	}
+	// Specifically query the FHIR endpoint metadata
+	u, err := url.Parse(urlString)
+	if err != nil {
+		// TODO: Use a logging solution instead of pringln
+		println("URL Parsing Error: ", err.Error())
+	}
+	u.Path = path.Join(u.Path, "metadata")
+
+	req, err := http.NewRequest("GET", u.String(), nil)
+	if err != nil {
+		// TODO: Use a logging solution instead of pringln
```

Comment:
`println` instead of `pringln`

Also, I prefer "TODO: Logging" and then we can just search for that string everywhere.

Reply:
Looks like `pringln` might be in several locations